tracer/src/Datadog.Trace/Ci/CiEnvironment/GitInfo.cs

```diff
@@ -156,13 +157,35 @@ public static IGitInfo GetCurrent()
             return null;
         }
 
-        var dirInfo = new DirectoryInfo(innerFolder);
+        DirectoryInfo? dirInfo;
+
+        try
+        {
+            dirInfo = new DirectoryInfo(innerFolder);
+        }
+        catch (Exception ex)
+        {
+            Log.Error(ex, "Error getting directory info");
```

Comment:
I don't think it's useful to report this error

```suggestion
            Log.Warning(ex, "Error getting directory info");
```